inference-engine/src/preprocessing/cpu_x86_avx512/ie_preprocess_gapi_kernels_avx512.cpp

```diff
@@ -125,26 +125,342 @@ void calcRowArea_32F(float dst[], const float *src[], const Size& inSz,
     calcRowArea_impl(dst, src, inSz, outSz, yalpha, ymap, xmaxdf, xindex, xalpha, vbuf);
 }
 
+static inline void verticalPass_lpi4_8U(const uint8_t *src0[], const uint8_t *src1[],
+                                        uint8_t tmp[], v_int16& b0, v_int16& b1,
+                                        v_int16& b2, v_int16& b3, v_uint8& shuf_mask,
+                                        int half_nlanes, int width) {
+    v_uint32 idxs1 = v_set_s32(23, 21, 7, 5, 22, 20, 6, 4, 19, 17, 3, 1, 18, 16, 2, 0);
```

Comment:
Can you left a comment about what these numbers stands for?

Reply:
Also can be done in several place below

Reply:
rename variables to more informative view